effects/zita1/zita.h

```diff
@@ -201,28 +218,27 @@ class ZitaReverb : public Effect
       {
       Q_OBJECT
 
-      float   _fsamp = 0.f;
+      float   _fsamp = 1.f;
```

Comment:
@RomanPudashkin another interesting moment. If we keep `0.f;` and object won't be initialized, than will catch division by zero;